### ISSUE TITLE
Failed to download Kiwi Browser

### ISSUE THREAD

**Issue by ElissonRodrigues** [Tobi823/ffupdater#250]
Stacktrace:
```
de.marmaro.krt.ffupdater.network.exceptions.InvalidApiResponseException: can't find release after all tries - abort
	at de.marmaro.krt.ffupdater.network.github.GithubConsumer.updateCheck(SourceFile:2)
	at de.marmaro.krt.ffupdater.network.github.GithubConsumer$updateCheck$1.invokeSuspend(Unknown Source:19)
	at w3.a.resumeWith(Unknown Source:8)
	at j4.i0.run(Unknown Source:100)
	at m4.d.run(Unknown Source:11)
	at n4.j.run(Unknown Source:2)
	at n4.a$a.run(SourceFile:82)
	Suppressed: j4.g0: [k1{Cancelling}@6084acf, Dispatchers.Main]
Timestamp: 2022-10-15T19:18:32.005

```
Device information:
| Key | Value |
| --- | --- |
| FFUpdater version | 77.3.0 (114) release |
| Device | SM-M515F (m51nsxx, m51, sm6150) |
| Manufacturer | samsung (samsung) | 
| Supported ABIs | arm64-v8a, armeabi-v7a, armeabi |
| Android version | 12 (SDK: 31) |
| OS | 21DK7A13, dpi, release-keys, 1661253216000 |

**Comment by HerrFrutti** [comment]
Same.

```
de.marmaro.krt.ffupdater.network.exceptions.InvalidApiResponseException: can't find release after all tries - abort
	at de.marmaro.krt.ffupdater.network.github.GithubConsumer.updateCheck(SourceFile:2)
	at de.marmaro.krt.ffupdater.network.github.GithubConsumer$updateCheck$1.invokeSuspend(Unknown Source:19)
	at w3.a.resumeWith(Unknown Source:8)
	at j4.i0.run(Unknown Source:100)
	at m4.d.run(Unknown Source:11)
	at n4.j.run(Unknown Source:2)
	at n4.a$a.run(SourceFile:82)
```

**Comment by HerrFrutti** [comment]
Also fails to download orbot...

Stacktrace:
```
de.marmaro.krt.ffupdater.network.exceptions.NetworkException: Fail to request the latest version of Orbot.
	at de.marmaro.krt.ffupdater.app.impl.Orbot.findLatestUpdate$ffupdater_release(SourceFile:161)
	at de.marmaro.krt.ffupdater.app.impl.Orbot$findLatestUpdate$1.invokeSuspend(Unknown Source:12)
	at w3.a.resumeWith(Unknown Source:8)
	at j4.i0.run(Unknown Source:100)
	at m4.d.run(Unknown Source:11)
	at n4.j.run(Unknown Source:2)
	at n4.a$a.run(SourceFile:82)
Caused by: de.marmaro.krt.ffupdater.network.exceptions.NetworkException: Fail to request the latest version of orbot from GitHub.
	at de.marmaro.krt.ffupdater.network.github.GithubConsumer.updateCheck(SourceFile:2)
	at de.marmaro.krt.ffupdater.network.github.GithubConsumer$updateCheck$1.invokeSuspend(Unknown Source:19)
	... 5 more
Caused by: de.marmaro.krt.ffupdater.network.exceptions.NetworkException: Fail to consume 'https://api.github.com/repos/guardianproject/orbot/releases/latest'.
	at de.marmaro.krt.ffupdater.network.ApiConsumer$consumeAsync$2$1.invokeSuspend(SourceFile:44)
	... 5 more
Caused by: de.marmaro.krt.ffupdater.network.exceptions.ApiRateLimitExceededException: API rate limit is exceeded.
	at de.marmaro.krt.ffupdater.network.ApiConsumer.consume(SourceFile:149)
	at de.marmaro.krt.ffupdater.network.ApiConsumer.access$consume(Unknown Source:0)
	at de.marmaro.krt.ffupdater.network.ApiConsumer$consumeAsync$2$1.invokeSuspend(SourceFile:20)
	... 5 more
Caused by: java.lang.Exception: response code is 403
	at de.marmaro.krt.ffupdater.network.ApiConsumer.consume(SourceFile:146)
	... 7 more
```
Device information:
| Key | Value |
| --- | --- |
| FFUpdater version | 77.3.0 (114) release |
| Device | Pixel 4a (sunfish, sunfish, sunfish) |
| Manufacturer | google (Google) | 
| Supported ABIs | arm64-v8a, armeabi-v7a, armeabi |
| Android version | 13 (SDK: 33) |
| OS | abfarm-release-rbe-64-00060, android-build, release-keys, 1661921084000 |

**Comment by Tobi823** [comment]
Fixed version (77.3.1) is now released
